rust/examples/config.ini

```diff
@@ -9,67 +9,95 @@
 
 # mqtt settings
 [mqtt]
-# the host is the broker to connect to
+# broker host to connect to
 host = test.mosquitto.org
-# the port is the port to connect to
+# broker port to connect to
 port = 8886
 # true to use the TLS protocol
 use_tls = true
-# false to use the MQTT WebSocket protocol
+# true to use the WebSocket protocol
 use_websocket = false
-# the client_id is the MQTT client id provided at the connection
-client_id = com_app_its-client
-# optional, for connection timeout
+# client id to provide at the connection
+client_id = com_app_its-client-1
+# optional, connection timeout
 connection_timeout = 60
-# optional, for ACL
+# optional, ACL username
 #username = username
-# optional, for ACL
+# optional, ACL password
 #password = password
 
 # geo_routing feature settings
 [geo]
-# the prefix is the prefix to use as MQTT topic root
+# prefix to use as MQTT topic root
 prefix = default
-# the suffix is the suffix to use as MQTT road id, e.g. v2x for a generic global map naming
+# the suffix to use as MQTT road id
 suffix = v2x
 
 # mobility feature settings
 [mobility]
+# UUID to use as source identifier
 source_uuid = com_app_its-client-1
+# station_id to use as source identifier
 station_id = 1
 # true to enable the responsibility
 use_responsibility = false
-# the number of threads to use
+# number of threads to use
 thread_count = 4
 
 # telemetry feature settings
 [telemetry]
-# the host is the telemetry server to connect to
+# telemetry server to connect to
 host = localhost
-# the port is the port to connect to
+#  telemetry port to connect to
 port = 4318
 # true to use the TLS protocol
 use_tls = false
-# optional, defaults to 'v1/traces'
+# optional, path of the request, default to 'v1/traces'
 #path = custom/v1/traces
```

Comment:
Not related to this PR, but we need to open an issue on this as for today this parameters defaults to `/v1/traces` but it as so requires the end user to also set `/v1/traces` when customizing the endpoint.

`/v1/traces` is actually some kind of a constant, so it should not be read from the configuration but hardcoded by the telemetry init function. And here we should only set the endpoint path if the target OTLP collector is routed so.

(this will also align implementation with the python one and maybe others)